src/cookbooks/analysis/data_catalog.md

```diff
@@ -0,0 +1,46 @@
+# The Data Catalog
+
+Reference material for data assets (tables, dashboards, pings, etc.) can primarily be found in the Data Catalog: https://mozilla.acryl.io.
+It provides an automatically updated "map" of data assets, including lineage and descriptions, without the need for manual curation.
+
+## What do I use it for?
+
+The primary use case for the catalog is finding out (at a glance) which data assets exist and how they relate to one another. A few examples:
+
+- Finding the source ping or table from a Looker dashboard.
+- Finding out whether a source ping or table has any downstream dependencies.
+- Getting a high-level overview of how tables are transformed before data shows up in a dashboard.
+- Tracing a column through various BigQuery tables.
+- Finding the source query or DAG that powers a particular BigQuery table.
+
+## How do I use it?
+
+Navigate to https://mozilla.acryl.io and log in via SSO. Once logged in, you should be able to explore assets via the
```

Comment:
Would an image be valuable here?

Reply:
The UI changes fairly frequently which would make this out of date quickly